### PR TITLE
Sum of primes in the inclusive range of entered number

### PR DESCRIPTION
**Pull request by Arundhati1Sharma**
General python code to find sum of primes upto an entered limit.

### CODE REVIEW

File: Basic-Scripts/Sum_of_prime.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+def is_prime(a): #function to check whether number is prime 
+    return all(a % i for i in range(2, a))
```
Comment (SKAUL05) [inline Basic-Scripts/Sum_of_prime.py]: Use meaningful variable names instead of alphabets

File: Basic-Scripts/Sum_of_prime.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        x = x+1
+    return sum
+
+def main():  #main function definition
+    print("Enter value upto which sum of primes to be calculated:")
+    n=int(input())
+    print(sum_of_primes(n))
+
+if __name__ == '__main__':  
+    main()
```
Comment (SKAUL05) [inline Basic-Scripts/Sum_of_prime.py]: Take input from here and call sum_of_primes from here too

File: Basic-Scripts/Sum_of_prime.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+def is_prime(a): #function to check whether number is prime 
+    return all(a % i for i in range(2, a))
+
+def sum_of_primes(n):
```
Comment (SKAUL05) [inline Basic-Scripts/Sum_of_prime.py]: Document your function using docstrings

**Review by SKAUL05 — CHANGES_REQUESTED** [review]

**Comment by Arundhati1Sharma** [comment]
Resolved requested changes.

**Review by Arundhati1Sharma — COMMENTED** [review]
Resolved requested changes..

File: Basic-Scripts/Sum_of_prime.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+def is_prime(is_a_prime):
+    '''Function 
+       to check whether the number is prime or not.'''
+    return all(is_a_prime % i for i in range(2, is_a_prime))
```
Comment (SKAUL05) [inline Basic-Scripts/Sum_of_prime.py:4]:
```suggestion
    return all(is_a_prime % element for element in range(2, is_a_prime))
```

File: Basic-Scripts/Sum_of_prime.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    is_a_prime = 2
+    while is_a_prime < number:
+        if is_prime(is_a_prime):
+            sum = sum + is_a_prime
+        is_a_prime = is_a_prime+1
+    return sum
+
+if __name__ == '__main__':  
+    print("Enter value upto which sum of primes to be calculated:")
+    number=int(input())
```
Comment (SKAUL05) [inline Basic-Scripts/Sum_of_prime.py:19]:
```suggestion
    number=int(input("Enter value upto which sum of primes to be calculated: "))
```

**Review by SKAUL05 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**